scripts/debug_tools/prepare_analyzer_cmd.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+import argparse
+import re
+import failure_lib as lib
+
+
+def getFirstLineOfFile(fname):
+    content = ''
+    with open(fname) as f:
+        content = f.readlines()
+    # this file has one line only
+    return content[0]
+
+
+# This is duplicated from the libcodechecer module
+# TODO could we have a common module for both libcodechecker and these debug
+# tools?
```

Comment:
I think it should be safe to reuse codechecker modules in this script.

Reply:
We had a conversation with @gyorb that these files should rather not be part of the package as this is not useful for the regular users.
We cannot just reuse modules from the package if we are not part of the package.

I am open to discuss again this decision, because being part of the package would solve this TODO.

Reply:
One reason I prefer to have this in the package it is possible to debug an old version of a codechecker just having the package without having to check out the corresponding version from the source control. It does not have to reside in the bin directory where other tools are.